eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -78,6 +78,8 @@ stages:
         jobs:
         - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
           parameters:
+            SparseCheckoutPaths:
+              - "sdk/${{ parameters.ServiceDirectory }}/**/*.json"
```

Comment:
I think it's worth adding a comment explaining that the `.js` based recordings are implicitly excluded. Because if someone in the future adds `PreGenerationSteps` that require a `js` file and adds that to the sparse checkout paths, then it will also end up checking out all the recordings.